src/index.html

```diff
@@ -66,7 +80,7 @@
     <div class="clearfix">
       <div class="pull-left text-muted" data-ng-controller="LanguageCtrl">
         <div uib-dropdown class="dropup">
-          <span>Language: </span>
+          <span>{{"LANGUAGE"|translate}}: </span>
```

Comment:
It has the additional ": " that is independent of translation. Probably not the best idea to include it in the locale data since that is error prone.